src/app/testing/gene-mocks.ts

```diff
@@ -90,101 +86,100 @@ export const geneMock1: Gene = {
       initial_nomination: 2018,
     },
   ],
-  medianexpression: [
+  median_expression: [
     {
       ensembl_gene_id: 'ENSG00000147065',
```

Comment:
The ticket notes that `medianExpression.ensembl_gene_id` was removed from `median_expression` -- should these mocks be updated to remove`ensembl_gene_id` from `median_expression`?

Reply:
yes, intellisense will demand it. 👍